catkin_ws/src/aquadrone_msgs/msg/SubState.msg

```diff
@@ -0,0 +1,8 @@
+geometry_msgs/Point position
+geometry_msgs/Vector3 pos_variance
+
+geometry_msgs/Vector3 velocity
+geometry_msgs/Vector3 vel_variance
+
+geometry_msgs/Quaternion orientation_quat
```

Comment:
Shouldn't we have orientation variance too?

Reply:
We should, yeah. I wasnt sure how to do it with a quaternion, but when I was making the Kalman Filter version, I saw some representation of orientation variance. So I'll try to add it in there, and propogate a merge into this.